cmd/kubeadm/app/constants/constants.go

```diff
@@ -56,4 +56,18 @@ const (
 	// Minimum amount of nodes the Service subnet should allow.
 	// We need at least ten, because the DNS service is always at the tenth cluster clusterIP
 	MinimumAddressesInServiceSubnet = 10
+
+	// DefaultTokenDuration specifies the default amount of time that a bootstrap token will be valid
+	DefaultTokenDuration = time.Duration(8) * time.Hour
+	// BootstrapTokenSecretPrefix the the prefix that will be used for the Secrets that are created as type bootstrap.kubernetes.io/token
+	BootstrapTokenSecretPrefix = "bootstrap-token-"
+
+	// CSVTokenBootstrapUser is currently the user the bootstrap token in the .csv file
+	// TODO: This should change to something more official and supported
+	// TODO: Prefix with kubeadm prefix
+	CSVTokenBootstrapUser = "kubeadm-node-csr"
```

Comment:
Oh, recalled incorrectly, yes `system:bootstrap:` as we decided in https://github.com/kubernetes/kubernetes/pull/41281